gef-ui/server/controllers/unissued-facilities/index.js

```diff
@@ -209,16 +208,21 @@ const postChangeUnissuedFacility = async (req, res) => {
     req.success = {
       message: `${body.facilityName} is updated`,
     };
-    req.url = `/gef/application-details/${dealId}/unissued-facilities`;
 
     // updates application with editorId
     const applicationUpdate = {
       editorId,
     };
     await api.updateApplication({ dealId, application: applicationUpdate, userToken });
 
-    // TODO: DTFS2-5227 change redirect
-    return applicationDetails(req, res);
+    // Change the redirect
```

Comment:
Comment doesn't make sense, and maybe is not required.

Reply:
Removed comment